lib/matplotlib/mathtext.py

```diff
@@ -2391,7 +2391,7 @@ def __init__(self):
         p.single_symbol <<= Regex(
             r"([a-zA-Z0-9 +\-*/<>=:,.;!\?&'@()\[\]|%s])|(\\[%%${}\[\]_|])" %
             unicode_range)
-        p.snowflake     <<= Suppress(p.bslash) + oneOf(self._snowflake)
+        p.accentprefixed<<= Suppress(p.bslash) + oneOf(self._accentprefixed)
```

Comment:
flake8 is not going to like this, I think

Reply:
We already suppress a whole lot of flake8 warnings in this file (because it looks better in this case to align things rather than equally space things).  Hoping this sneaks by under that to avoid thrashing a whole lot of lines.

Reply:
nope, flake8 did care.  Accepted a bit of raggedness in the `<<=` lining up rather than touching every line.